lms/djangoapps/instructor/views/data_access.py

```diff
@@ -85,7 +98,7 @@ def get_group_query_students(course_id, group_id):
         queried_students = retrieve_grouped_query(course_id, group_id)
         return queried_students
     # return empty otherwise
-    return StudentsForQuery.objects.none()
+    return User.objects.none()
```

Comment:
Not related to the change User here, but I feel like the if check for subqueries is redundant, if there aren't any, it will be handled in retrieve_grouped_query anyways. I think the original was cleaner: https://github.com/Stanford-Online/edx-platform/commit/a29f489c0670b9c11ca8666d07e6be59a3a9dccc#diff-5bcc942b15d192b641ea0e2ef95b2167R75